managed/utils/encryption/helpers.go

```diff
@@ -31,6 +31,33 @@ import (
 	"gopkg.in/reform.v1"
 )
 
+func encryptionKeyPath() string {
+	customKeyPath := os.Getenv("PMM_ENCRYPTION_KEY_PATH")
+	if customKeyPath != "" {
+		return customKeyPath
+	}
+
+	return DefaultEncryptionKeyPath
+}
+
+func backupOldEncryptionKey() error {
+	err := os.Rename(encryptionKeyPath(), fmt.Sprintf("%s_old.key", strings.TrimSuffix(encryptionKeyPath(), ".key")))
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (e *Encryption) backupOldEncryptionKey() error {
+	err := os.Rename(e.Path, fmt.Sprintf("%s_old.key", strings.TrimSuffix(e.Path, ".key")))
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
```

Comment:
do we really need this duplication?

Reply:
I prepared it because for all methods before we had method and its wrap around default. Removed for newly added methods.